arangod/Scheduler/SupervisedScheduler.cpp

```diff
@@ -104,11 +101,17 @@ SupervisedScheduler::SupervisedScheduler(uint64_t minThreads,
 
 SupervisedScheduler::~SupervisedScheduler() {}
 
+bool SupervisedScheduler::queue(RequestLane lane, std::function<void()> const& handler) {
+  std::function<void()> copy = handler;
+  return queue(lane, std::move(copy));
```

Comment:
You could change the signature of `queue` to take `handler` per value, then you would't need a separate overload - r-values would be moved, l-values (like const refs) would be copied.